docs/language/functions/grok.md

```diff
@@ -0,0 +1,44 @@
+### Function
+
+&emsp; **grok** &mdash; parse a string using a grok pattern
+
+### Synopsis
+
+```
+grok(pattern: string, s: string) -> any
+grok(extra: string, pattern: string, s: string) -> any
```

Comment:
It's surprising to see `extra` before rather than after the parameters common to both forms. Any reason not to move it after `s`?

Reply:
ok